.travis.yml

```diff
@@ -10,9 +10,17 @@ before_install:
   - redis-server --version
 install:
   - pip install --upgrade pip
-  - pip install mock nose>=0.10.1 pep8 flake8 coveralls
+  - pip install mock nose>=0.10.1 pep8 flake8 coveralls click natsort
   - travis_retry pip install .
 script:
+  - git clone https://github.com/biocore/american-gut-web.git ~/build/biocore/american-gut-web
+  - cd /home/travis/build/biocore/american-gut-web
+  - export AG_CONFIG=`pwd`/ag_config.txt.example
+  - export PYTHONPATH=/home/travis/build/biocore/american-gut-web:$PYTHONPATH
+  - ./scripts/ag make test
+  - export PYTHONPATH=
+  - $PYTHONPATH
+  - cd ~/build/biocore/labadmin/knimin/
```

Comment:
There should be some magical travis env variable for this. See [here](http://docs.travis-ci.com/user/environment-variables/) for a list